pkg/apis/intelligence/v1alpha1/types.go

```diff
@@ -41,14 +41,14 @@ type NetworkPolicyRecommendation struct {
 }
 
 type NetworkPolicyRecommendationStatus struct {
-	State                    string      `json:"state,omitempty"`
-	SparkApplication         string      `json:"sparkApplication,omitempty"`
-	CompletedStages          int         `json:"completedStages,omitempty"`
-	TotalStages              int         `json:"totalStages,omitempty"`
-	RecommendedNetworkPolicy string      `json:"recommendedNetworkPolicy,omitempty"`
-	ErrorMsg                 string      `json:"errorMsg,omitempty"`
-	StartTime                metav1.Time `json:"startTime,omitempty"`
-	EndTime                  metav1.Time `json:"endTime,omitempty"`
+	State                 string      `json:"state,omitempty"`
+	SparkApplication      string      `json:"sparkApplication,omitempty"`
+	CompletedStages       int         `json:"completedStages,omitempty"`
+	TotalStages           int         `json:"totalStages,omitempty"`
+	RecommendationOutcome string      `json:"recommendationOutcome,omitempty"`
```

Comment:
forgive me for this dumb question, but should RecommendationOutcome be also int he CRD descriptor?

Reply:
I think this is the intelligence APIs instead of a CRD. We need this because it defines the contents in the response to CLI.

Reply:
My question was about whether we needed this attribute must be explicitly specified in the CRD as well. We clarified this is not the case, thanks!